tests/common/test_VanDerCorput.cpp

```diff
@@ -216,3 +216,37 @@ TEST(VanDerCorput, IterationEndsWhenMinimumResolutionReached)
   EXPECT_EQ(7, iterations);
   EXPECT_EQ(7, vdc_0_125.getLength());
 }
+
+TEST(VanDerCorput, ScaleAndIterateProperlyOverSpanWithResolution)
+{
+  // Check that with a non-unit span, iteration stops appropriately.
+  VanDerCorput vdc{2, false, false, 0.25};
+  int iterations = 0;
+  EXPECT_NO_THROW({
+    for (VanDerCorput::const_iterator itr = vdc.begin(); itr != vdc.end();
+         ++itr)
+    {
+      ++iterations;
+    }
+  });
+  EXPECT_EQ(7, iterations);
+  EXPECT_EQ(7, vdc.getLength());
+
+  // Check that with a non-unit span, the vdc values are correct.
+  EXPECT_DOUBLE_EQ(1.00, vdc[0].first);
+  EXPECT_DOUBLE_EQ(0.50, vdc[1].first);
+  EXPECT_DOUBLE_EQ(1.50, vdc[2].first);
+  EXPECT_DOUBLE_EQ(0.25, vdc[3].first);
+  EXPECT_DOUBLE_EQ(1.25, vdc[4].first);
```

Comment:
This isn't especially critical, but I would've expected the third level of the tree to be 0.25, 0.75, 1.25, 1.75 from the documentation. Why isn't that what we return?

Reply:
Where does it say that? That would be wrong, sequence jumps around: https://en.wikipedia.org/wiki/Van_der_Corput_sequence

Reply:
I was expecting the same or for it to be 0.25, 1.75, 0.75, 1.25 until I wrote the test and learned my lesson.

Reply:
Ah, I see. I was slightly mislead by [level-order traversal](https://github.com/personalrobotics/aikido/blob/84d78563360a86457bc220682c10fec1937b2f12/include/aikido/common/VanDerCorput.hpp#L14). It's technically correct (in that we're processing the whole level before advancing to the next one), but I've previously seen it as also processing each level from left to right.

I think it's fine as is to provide intuition for what the sequence does (and not a critical distinction that needs to be fixed).